_pkgdown.yml

```diff
@@ -34,14 +38,13 @@ reference:
   - use_lintr
   - xml_nodes_to_lints
 
-- title: internal
+- title: Internal
```

Comment:
internal also doesn't quite feel right... Meta-tooling?

Reply:
Note that this is just to mark functions which are internal and should not to be displayed in [the reference index](https://lintr.r-lib.org/reference/index.html).

And, as expected, it doesn't show up in the TOC or in the index:

<img width="306" alt="Screenshot 2022-07-31 at 09 13 57" src="https://user-images.githubusercontent.com/11330453/182014577-d268dd9c-1c80-4577-a6e9-c36adecd17a1.png">

Reply:
But I have still renamed that section to `Meta-tooling`.

Reply:
oh, I see. indeed I don't see any reason to hide exported functions from the site. thanks!